lib/handler/mruby.c

```diff
@@ -329,6 +312,25 @@ static h2o_mruby_shared_context_t *get_shared_context(h2o_context_t *ctx)
     return *data;
 }
 
+mrb_value prepare_fibers(h2o_mruby_context_t *ctx)
+{
+    mrb_state *mrb = ctx->shared->mrb;
+
+    h2o_mruby_config_vars_t config = ctx->handler->config;
+    mrb_value conf = mrb_hash_new_capa(mrb, 3);
+    mrb_hash_set(mrb, conf, mrb_symbol_value(mrb_intern_lit(mrb, "code")), mrb_str_new(mrb, config.source.base, config.source.len));
+    mrb_hash_set(mrb, conf, mrb_symbol_value(mrb_intern_lit(mrb, "file")), mrb_str_new(mrb, config.path, strlen(config.path)));
+    mrb_hash_set(mrb, conf, mrb_symbol_value(mrb_intern_lit(mrb, "line")), mrb_fixnum_value(config.lineno));
+
+    /* run code and generate handler */
+    mrb_value result =
+        mrb_funcall(mrb, mrb_obj_value(mrb->kernel_module), "_h2o_prepare_app", 1, conf);
+    h2o_mruby_assert(mrb);
```

Comment:
@kazuho Thank you for the clarification, I understand what you meant.
I rechecked `on_context_init`, then I came to believe that `_h2o_prepare_fibers` itself doesn't do nothing without creating fibers, so we need not to check exceptions and report to the users. So I actually remove exception check at https://github.com/h2o/h2o/pull/1173/commits/e592dcac51d4ca5d0b3376aa1288f51bd805801c. How's this?